hw/dv/tools/rules.mk

```diff
@@ -47,10 +47,10 @@ ifneq (${SW_NAME},)
 	ninja -C $(SW_BUILD_DIR)/build-out sw/device/$(SW_DIR)/$(SW_NAME)_export_$(SW_BUILD_DEVICE)
 
 	mkdir -p $(SW_BUILD_DIR)/sw $(SW_BUILD_DIR)/rom
-	cp $(SW_BUILD_DIR)/build-out/sw/device/boot_rom/boot_rom_$(SW_BUILD_DEVICE).vmem \
-		$(SW_BUILD_DIR)/rom/rom.vmem
-	cp $(SW_BUILD_DIR)/build-out/sw/device/$(SW_DIR)/$(SW_NAME)_$(SW_BUILD_DEVICE).vmem \
-		$(SW_BUILD_DIR)/sw/sw.vmem
+	cp $(SW_BUILD_DIR)/build-out/sw/device/boot_rom/boot_rom_$(SW_BUILD_DEVICE).32.vmem \
+		$(SW_BUILD_DIR)/rom/rom.32.vmem
+	cp $(SW_BUILD_DIR)/build-out/sw/device/$(SW_DIR)/$(SW_NAME)_$(SW_BUILD_DEVICE).32.vmem \
+		$(SW_BUILD_DIR)/sw/sw.32.vmem
```

Comment:
You can revert the changes in this file. It is a part of the old Makefile flow which is already broken in other places. We do not plan to support it going forward. It will be removed eventually.